src/plugins/codeStyle/index.ts

```diff
@@ -125,6 +128,17 @@ export default class CodeStyle {
                     diagnostics.push(messages.noPrint(s.tokens.print.range, noPrint));
                 }
             },
+            LiteralExpression: e => {
+                if (validateColorStyle && e.token.kind === TokenKind.StringLiteral) {
+                    validateColorStyle(e.token.text, e.token.range, diagnostics);
+                }
+            },
+            TemplateStringExpression: e => {
+                const token = e.quasis[0].expressions[0].token;
```

Comment:
Are all the expressions always string literals?

Also this expects the very first expression to be provided - I suspect it can break in a number of ways:
- empty template string,
- complex template like `${value}something`.

Probably you shouldn't check any case of template string with zero or more than one expression, and you need to ensure the 1st expression is a string. No?

cc @TwitchBronBron

Reply:
Empty template string as in ``?